pixels/pixels.py

```diff
@@ -70,20 +47,6 @@ def custom_openapi() -> t.Dict[str, t.Any]:
         version="1.0.0",
```

Comment:
Bump the version in here too, since it was bumped in `pyproject.toml`
```suggestion
        version="2.0.0",
```